src/Transport/Creation/AzureServiceBusQueueCreator.cs

```diff
@@ -40,6 +43,28 @@ public AzureServiceBusQueueCreator(ReadOnlySettings settings)
             }
         }
 
+        void TryGetSystemQueuessAddresses()
```

Comment:
Agree with @andreasohlund 
Sending destinations should not be created. We should also stop doing it https://github.com/Particular/NServiceBus/blob/release-6.0.0/src/NServiceBus.Core/Transports/Msmq/QueueCreator.cs#L23 but of course this has other implications for F5ability